openmc/model/surface_composite.py

```diff
@@ -748,9 +748,10 @@ class XConeOneSided(CompositeSurface):
     z0 : float, optional
         z-coordinate of the apex. Defaults to 0.
     r2 : float, optional
-        Parameter related to the aperture [:math:`\\rm cm^2`].
-        It can be interpreted as the increase in the radius squared per cm along
-        the cone's axis of revolution.
+        Parameter related to the aperture [:math:`\\rm cm^2`]. This is the
+        square of the radius of the cone 1 cm from the apex. This can also be
+        treated as the square of the cone's slope relative to its axis of
+        revolution. Defaults to 1.
```

Comment:
``` suggestion
This parameter is the square of the slope of the cone.
It is defined as :math:`\left(\frac{r}{h}\right)^2` for a radius, :math:`r`, a distance :math:`h` from the apex.
An easy way to define this quantity is to take the square of the radius of the cone (in cm) 1 cm from the apex.
```

Reply:
I think this is a great description with a small change defining 'h' as the 'axial distance' 
```suggestion
        This parameter is the square of the slope of the cone.
        It is defined as :math:`\left(\frac{r}{h}\right)^2` for a radius, :math:`r`, an axial distance :math:`h` from the apex.
        An easy way to define this quantity is to take the square of the radius of the cone (in cm) 1 cm from the apex.
```

Reply:
I like it. Thanks for the iterations @MicahGale, @NybergWISC!